blueprint/index.md

```diff
@@ -93,213 +96,219 @@ For more information about managing variables can be found here: <https://help.m
+- ***Case 2*** - The Expert has expressed an opinion they believe the conversation has been completed and this event is detected via the Limitless update type = ‘cust\_confirmation’. If a CSAT has not been sent, then this event triggers a CSAT link (shortened via a Data Action) to be sent to the customer.
 
 ![](images/029.png)
 
-- ***Case 3*** - A Limitless SmartCrowd status change event is detected via the Limitless update type = ‘state’. These status changes are further evaluated, and the following actions performed:
-  - ***‘Escalated’*** - The Expert Crowd have decided they cannot resolve this question. When this state is detected, the Customer receives a message stating that an Agent will be brought into the conversation and a ‘Transfer to ACD’ block is used to bring in the Agent.
-  - ***‘TimedOut’*** - The Expert Crowd did not respond in time. When this state is detected, the Customer receives a message stating that an Agent will be brought into the conversation and a ‘Transfer to ACD’ block is used to bring in the Agent.
-  - ***‘Open’ & Not ‘TimedOut’*** - The Limitless lifecycle is in a state where a CSAT, if not already issued, should be sent to the Customer. The flow actions this.
-  - ***‘Resolved’*** - The Limitless lifecycle has completed and the question was resolved. This is an end state and the flow disconnects.
+- ***Case 3*** - A Limitless SmartCrowd status change event is detected via the Limitless update type = ‘state’. These status changes are further evaluated, and the following actions are performed:
+  - ***‘Escalated’*** - The Expert Crowd has decided they cannot resolve this question. When this state is detected, the customer receives a message stating an Agent will be brought into the conversation, and a ‘Transfer to ACD’ block is used to bring in the Agent.
+  - ***‘TimedOut’*** - The Expert Crowd did not respond in time. When this state is detected, the customer receives a message stating an Agent will be brought into the conversation, and a ‘Transfer to ACD’ block is used to bring in the Agent.
+  - ***‘Open’ & Not ‘TimedOut’*** - The Limitless lifecycle is in a state where a CSAT, if not already issued, should be sent to the customer. The flow actions this.
+  - ***‘Resolved’*** - The Limitless lifecycle was completed, and the question resolved. This is an end state, and the flow disconnects.
 
 ![](images/030.png)
-## **Cloud Analytics & Reporting**
+## **Cloud analytics & reporting**
 The use of the Limitless flows has no impact on Cloud Analytics or Genesys reporting.
-## **Other Considerations**
+
+## **Other considerations**
 - ***Flow Outcomes*** - The blueprint does not contain Flow Outcomes to avoid validation errors on import. You should consider what Flow Outcomes you want and introduce these to Flow.
```

Comment:
I usually like to end with a Resources section that provides a link to any websites or URLs we referenced in the document.  (Not necessarily the flows, but any API or additional docs.